doc/user/layouts/shortcodes/system-variables.html

```diff
@@ -1,7 +1,12 @@
 ## System variables
 
-Materialize reserves system variables for region-wide configuration. Although it's possible to `SHOW` system variables, you must [contact us](https://materialize.com/contact/) to set the value of most variables (e.g. increasing the maximum number of AWS PrivateLink connections in your region).
-Some system variables can be modified by any superuser (i.e. Frontegg admin).
+Materialize reserves system variables for region-wide configuration. Although it's possible to
+`SHOW` system variables, you must [contact us](https://materialize.com/contact/) to set the value
+of most variables (e.g. increasing the maximum number of AWS PrivateLink connections in your
+region), but some system variables can be modified by any superuser (i.e. Frontegg admin).
+
+A subset of [session variables](../set/#session-variables) can also have their region-wide default
```

Comment:
I find this confusing: you can set the default for _session_ variables using `ALTER SYSTEM`? Tried it out as an admin, and got:

```sql
ALTER SYSTEM SET cluster = 'other_cluster';

Error: permission denied to alter system
Detail: You must be the 'mz_system' role
```